packages/@sanity/core/src/actions/graphql/getSanitySchema.js

```diff
@@ -30,6 +30,7 @@ function provideFakeGlobals() {
 
 function getSanitySchema(basePath) {
   const domCleanup = jsdomGlobal()
+  const windowCleanup = () => global.window.close()
   const globalCleanup = provideFakeGlobals()
```

Comment:
The only thing I could think of is that if this cleanup fits in the `globalCleanup`?